packages/labs/testing/src/lib/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+export const nextFrame = () =>
+  new Promise((resolve) => requestAnimationFrame(resolve));
+
+/**
+ * Strips Lit expression comments from provided html string.
+ */
+export const stripExpressionComments = (html: string) =>
```

Comment:
I think we should have some tests for `stripExpressionComments` and `stripExpressionMarkers` probably. Can be an issue/follow up PR though.

Reply:
Filed https://github.com/lit/lit/issues/3102